src/Spryker/Zed/SynchronizationBehavior/Persistence/Propel/Behavior/SynchronizationBehavior.php

```diff
@@ -184,6 +196,17 @@ public function modifyTable()
             }
         }
 
+        if (!$table->hasColumn('alias_keys')) {
+            $table->addColumn([
+                'name' => 'alias_keys',
+                'type' => 'VARCHAR',
+            ]);
+            $uniqueIndex = new Unique();
+            $uniqueIndex->setName($table->getName() . '-unique-alias-keys');
+            $uniqueIndex->addColumn($table->getColumn('alias_keys'));
```

Comment:
This is a major

Reply:
Please make this as an optional column which be added by a new parameter to `synchronization-behavior`, so the current projects can update this module in BC way.